std/container/array.d

```diff
@@ -2111,23 +2112,22 @@ if (is(Unqual!T == bool))
             _store._backend.length = _store._backend.length - 1;
         }
     }
+
     /// ditto
     alias stableRemoveBack = removeBack;
```

Comment:
Wait, I think it is because of documentation - this way both aliases are documented accordingly. So it should stay.